src/deepsparse/server/server.py

```diff
@@ -208,42 +206,32 @@ def _add_endpoint(
     endpoint_config: EndpointConfig,
     executor: ThreadPoolExecutor,
     context: Context,
-    pipeline_logger: ManagerLogger,
+    server_logger: BaseLogger,
 ):
     pipeline_config = endpoint_config.to_pipeline_config()
     pipeline_config.kwargs["executor"] = executor
 
     _LOGGER.info(f"Initializing pipeline for '{endpoint_config.name}'")
-    pipeline = Pipeline.from_config(pipeline_config, context)
+    pipeline = Pipeline.from_config(pipeline_config, context, server_logger)
 
     _LOGGER.info(f"Adding endpoints for '{endpoint_config.name}'")
-    _add_pipeline_endpoint(
-        app, endpoint_config, pipeline, pipeline_logger, server_config.integration
-    )
+    _add_pipeline_endpoint(app, endpoint_config, pipeline, server_config.integration)
 
 
 def _add_pipeline_endpoint(
     app: FastAPI,
     endpoint_config: EndpointConfig,
     pipeline: Pipeline,
-    pipeline_logger: ManagerLogger,
     integration: str = INTEGRATION_LOCAL,
 ):
     input_schema = pipeline.input_schema
     output_schema = pipeline.output_schema
-    pipeline_name = pipeline.alias or pipeline.task
 
     def _predict(request: pipeline.input_schema):
-        (
-            pipeline_outputs,
-            pipeline_inputs,
-            engine_inputs,
-            inference_timing,
-        ) = pipeline.run_with_monitoring(request)
-        pipeline_logger.log_latency(pipeline_name, inference_timing)
-        pipeline_logger.log_data(
-            pipeline_name, (pipeline_inputs, engine_inputs), pipeline_outputs
-        )
+        pipeline_outputs = pipeline(request)
+        server_logger = pipeline.logger
+        if server_logger:
+            log_system_info(server_logger)
```

Comment:
fine for now, i think we may want to move this into a background thread in the future. e.g. if server doesn't receive a request for an hour we still probablky want to be logging system info during that time

Reply:
+1 that is a great callout